apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,339 @@
+            // Make the request
+            CheckAndMutateRowResponse response = bigtableClient.CheckAndMutateRow(
+                tableName,
+                rowKey,
+                predicateFilter,
+                trueMutation1, trueMutation2);
+
+            if (response.PredicateMatched)
+            {
+                // The true mutations were applied...
+            }
+            else
+            {
+                // The true mutations were not applied...
+            }
+            // End snippet
+
+            // TODO: Verifications
+        }
+
+        public void MutateRow()
+        {
+            // Snippet: MutateRow(TableName,RowKey,IEnumerable<Mutation>,CallSettings)
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            RowKey rowKey = new RowKey(12, 255, 0, 17, 1);
+            IEnumerable<Mutation> mutations = new Mutation[]
+            {
+                Mutations.DeleteFromFamily("cf1"),
+                Mutations.DeleteFromColumn("cf2", "c1", new BigtableVersionRange(null, new BigtableVersion(DateTime.UtcNow)))
+            };
+            // Make the request
+            MutateRowResponse response = bigtableClient.MutateRow(
+                tableName,
+                rowKey,
+                mutations);
+            // End snippet
+
+            // TODO: Verifications
+        }
+
+        public void MutateRowWithParams()
+        {
+            // Snippet: MutateRow(TableName,RowKey,Mutation[])
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            RowKey rowKey = new RowKey(12, 255, 0, 17, 1);
+            Mutation mutation1 = Mutations.DeleteFromFamily("cf1");
+            Mutation mutation2 =
+                Mutations.DeleteFromColumn("cf2", "c1", new BigtableVersionRange(null, new BigtableVersion(DateTime.UtcNow)));
+            // Make the request
+            MutateRowResponse response = bigtableClient.MutateRow(
+                tableName,
+                rowKey,
+                mutation1, mutation2);
+            // End snippet
+
+            // TODO: Verifications
+        }
+
+        public async Task MutateRows()
+        {
+            // Snippet: MutateRows(TableName,IEnumerable<MutateRowsRequest.Types.Entry>,CallSettings)
+            // Create client
+            BigtableClient bigtableClient = BigtableClient.Create();
+            // Initialize request argument(s)
+            TableName tableName = new TableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+            IEnumerable<MutateRowsRequest.Types.Entry> entries = new MutateRowsRequest.Types.Entry[]
+            {
+                Mutations.CreateEntry("r1",
```

Comment:
Each `Entry` is an atomic, ordered set of mutations for a particular row. `Mutations.CreateEntry` takes the key of the row to act upon and then the mutations to use. Perhaps this should have a different name to make it clearer. `Mutations.RowMutationsEntry` perhaps?

Also, this doesn't need to live on `Mutations`. I just didn't want to add static methods to the nested `MutateRowsRequest.Types.Entry` class and `Entries` sounded too generic to follow the naming convention I've been using here. I'm open to suggestions for where else this could live.